src/SonarQube.Client.Tests/Requests/Api/V7_20/GetIssuesRequestWrapperTests.cs

```diff
@@ -34,10 +35,15 @@ namespace SonarQube.Client.Tests.Requests.Api.V7_20
     [TestClass]
     public class GetIssuesRequestWrapperTests
     {
-        [TestMethod]
-        public async Task InvokeAsync_NoIssueKeys_ExpectedPropertiesArePassedInMultipleRequests()
+        private const string SonarQubeComponentPropertyName = "components";
+        private const string SonarCloudComponentPropertyName = "componentKeys";
```

Comment:
Personal preference: I would name them like this. The common prefix makes it more table-layouted and easier to read (easier to read what's the common part)
```suggestion
        private const string ComponentPropertyNameSonarQube = "components";
        private const string ComponentPropertyNameSonarCloud = "componentKeys";
```